instrumentation/pekko-actor-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pekkoactor/PekkoActorCellInstrumentation.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.pekkoactor;
```

Comment:
our current convention is to also add the version number to package name `io.opentelemetry.javaagent.instrumentation.pekkoactor.v1_0`

Reply:
I've now done that for pekkoactor and pekkohttp, for both the `src/main` and `src/test` packages.